### ISSUE TITLE
Page redirection

### ISSUE THREAD

**Issue by FranklinYu** [claudioc/jingo#252]
This is a feature request.

When we rename a page, previous link to the old URL would be dead. It would be great if we can edit the old page with things like this:

```text
#REDIRECT [[new page]]
```

And it automatically redirects visitors to the new page.

**Comment by claudioc** [comment]
Yes, that would be a great addition, but Jingo doesn't have a database so it's hard to image the correct place to save those redirects... maybe embedding the information directly in the page itself? I don't know. I will leave it open for Jingo 2 :)

**Comment by FranklinYu** [comment]
I would suggest to save it as normal file. When Jingo render it, if the file contains the magic words, return a redirection instead of “200 OK” rendering the words.
